test/saithrift/tests/sail3.py

```diff
@@ -1324,4 +1325,159 @@ def runTest(self):
             attr = sai_thrift_attribute_t(id=SAI_PORT_ATTR_PORT_VLAN_ID, value=attr_value)
             self.client.sai_thrift_set_port_attribute(port1, attr)
             self.client.sai_thrift_set_port_attribute(port2, attr)
-
+@group('lag')            
+@group('l3')
+class L3MultipleLagTest(sai_base_test.ThriftInterfaceDataPlane):
+v4_enabled = 1
+    v6_enabled = 1
+    ip_mask = '255.255.255.0'
+    addr_family = SAI_IP_ADDR_FAMILY_IPV4
+    lag_members = []
+    lags = []
+    lags_rifs = []
+    neighbors = []
+    routes = []
+    vr_id = 0
```

Comment:
can we add total_lag_port = 16, and use total_lag_port in the reset of the program instead of hardcoded number 16?